spec/features/index_page_spec.rb

```diff
@@ -22,6 +22,15 @@
     expect(page).to have_content(customer.email)
   end
 
+  it "links to the customer show page without javascript", js: false do
```

Comment:
I wonder if this means that now we'll be able to disable JavaScript by default and _enable_ JS instead?